framework/src/st_things/things_stack/src/common/framework/things_data_manager.c

```diff
@@ -864,7 +864,7 @@ static int parse_things_info_json(const char *filename)
 				cJSON *spec_platform = cJSON_GetObjectItem(specification, KEY_DEVICE_SPECIFICATION_PLATFORM);
 				if (NULL != spec_platform) {
 					cJSON *manufacturer_name = cJSON_GetObjectItem(spec_platform, KEY_DEVICE_SPECIFICATION_PLATFORM_MANUFACTURERNAME);
-					cJSON *manufacturer_uri = cJSON_GetObjectItem(spec_platform, KEY_DEVICE_SPECIFICATION_PLATFORM_MANUFACTURERURI);
+					cJSON *manufacturer_url = cJSON_GetObjectItem(spec_platform, KEY_DEVICE_SPECIFICATION_PLATFORM_MANUFACTURERURI);
```

Comment:
2nd argument of function has URI, not URL. What is right? If URL is right, you should modify MACRO also.